data/yast2/firstboot/firstboot_custom.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0"?>
```

Comment:
Problem to solve now with this custo xml. I took it from the package for leap 15.2. On TW, this file now contain this:
`	<!-- This option is deprecated in favor of installation-layout -->
	<!-- <installation_ui>sidebar</installation_ui> -->

	<!-- Configuration of the installation/firstboot layout -->
	<installation_layout>
		<mode>steps</mode>
		<banner config:type="boolean">false</banner>
	</installation_layout>`

Still works, but then we don't have the side menu check [with TW's control file](http://waaa-amazing.suse.cz/tests/12825#step/yast2_firstboot/1) versus [Leap 15.2's file on TW](http://waaa-amazing.suse.cz/tests/12838#step/yast2_firstboot/1). On SP3, the side menu appears with this profile.

So, 2 solutions
1 - create a new set of needles without the side menu for TW
2 - add the xml section above to the custom file. Shows the side menu on both TW and SP3

Reply:
it was announced that new feature of having those two modes a few weeks ago, but we should test using the default for each product and modification on top of the default so it is ok duplication if you need it.